about_asserts.exs

```diff
@@ -36,6 +37,7 @@ defmodule AboutAsserts do
     assert_raise ExUnit.AssertionError, message, is_1_greater_than_2?
   end
 
+  @tag :skip
```

Comment:
The solution doesn't handle `assert_?` yet. Will have to figure that out and remove all these skips.